tests/utils.py

```diff
@@ -8,9 +8,16 @@
 
 from conda_build.conda_interface import PY3
 from conda_build.metadata import MetaData
-from conda_build.utils import on_win, FileNotFoundError
+from conda_build.utils import on_win
 
-thisdir = os.path.dirname(os.path.realpath(__file__))
+
+def get_root_dir():
+    import conda_build
+    conda_build_dir = os.path.realpath(os.path.dirname(conda_build.__file__))
+    return os.path.abspath(os.path.join(conda_build_dir, ".."))
```

Comment:
temporary fix, that will be replaced in a second step by fixtures to handle it properly